inc/customizer/custom-controls/color-group/color-group.scss

```
@@ -0,0 +1,27 @@
+.customize-control.customize-control-ast-color-group {
```

```suggestion
.customize-control
	+.customize-control-ast-color-group {
```

I believe in SCSS we should be writing it this way?

---

Done